### PR TITLE
ci: add some tests

### LINKED ISSUE #3: ci: add workflow
- build
- lint
- test
- ~~publish~~

### PR DESCRIPTION
**Pull request by anbraten**
Closes #3

### CODE REVIEW

**Comment by anbraten** [comment]
@lukashass Any idea why the service type is broken?

**Comment by lukashass** [comment]
> @lukashass Any idea why the service type is broken?

No but there is already an issue for that #1

File: package.json [hunk 1]
```diff
@@ -20,9 +20,11 @@
...
     "build": "rollup -c rollup.config.js",
     "clean": "rm -rf dist/ node_modules/",
-    "lint": "prettier --check . && eslint --max-warnings 0 .",
+    "lint": "eslint --max-warnings 0 .",
+    "lint:format": "prettier --check .",
     "start": "rollup -c rollup.config.js --watch",
-    "test": "NODE_OPTIONS=--experimental-vm-modules jest --forceExit --detectOpenHandles",
+    "test": "NODE_OPTIONS=--experimental-vm-modules jest --forceExit --detectOpenHandles --testLocationInResults --json --outputFile=report.json",
+    "typecheck": "tsc --noEmit",
     "test:watch": "yarn test --watch",
```
Comment (lukashass) [inline package.json]:
```suggestion
    "test:watch": "yarn test --watch",
    "typecheck": "tsc --noEmit",
```

**Review by lukashass — DISMISSED** [review]

File: package.json [hunk 2]
```diff
@@ -46,7 +48,6 @@
     "esbuild-register": "2.6.0",
```
Comment (lukashass) [inline package.json]: @anbraten I think this can be removed

**Review by lukashass — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (lukashass) [reply package.json:48]:
```suggestion
```

Comment (anbraten) [reply package.json]: @lukashass removed it

**Review by lukashass — APPROVED** [review]

**Pull request merged**